src/utils/error/player-error.js

```diff
@@ -0,0 +1,55 @@
+//@flow
+import getLogger, {getLogLevel, LogLevel} from '../logger'
+import {Severity} from './severity'
+import {Code} from './code'
+import {Category} from './category'
+
+
+const ERROR_LOGGER: string = 'Error';
+/**
+ * @classdesc This is a description of the error class.
+ */
+export default class Error {
+  UNKNOWN: string = 'UNKNOWN';
+  severity: number;
+  category: number;
+  code: number;
+  data: any;
+  /**
+   * @enum {number}
+   */
+  static Severity: { [level: string]: number } = Severity;
```

Comment:
why u called it `level:` in the type?
it should be { [severity: string]: number }